mock-plugin/src/main/resources/mock-idv-user-story.json

```diff
@@ -1,6 +1,6 @@
 {
   "scenes":[
-    {"frameNumber" : 0, "stepCode" : "START", "step" : { "code" : "liveness_check", "framesPerSecond" : 3, "durationInSeconds" : 100, "startupDelayInSeconds" : 2, "retryOnTimeout" : false, "retryableErrorCodes" : [] }, "feedback" : null },
+    {"frameNumber" : 0, "stepCode" : "START", "step" : { "code" : "liveness_check", "framesPerSecond" : 1, "durationInSeconds" : 100, "startupDelayInSeconds" : 2, "retryOnTimeout" : false, "retryableErrorCodes" : [] }, "feedback" : null },
```

Comment:
https://github.com/mosip/esignet-plugins/blob/develop/mock-plugin/src/main/resources/mock-idv-user-story.json#L3 This change also exists. Kindly cross-check.